### PR TITLE
Update migration instructions to take into account new ISOs and PTF files

### PR DESCRIPTION
**Pull request by cadenzajon**

### CODE REVIEW

File: xml/art_migration.xml [hunk 1]
```diff
@@ -834,22 +835,42 @@ ansible-playbook -i hosts/verb_hosts ardana-user-create.yml</screen>
...
-     Begin the Ardana.Pike installation:
+     Begin the installation:
     </para>
 <screen>sudo zypper -n in patterns-cloud-ardana
 sudo su - ardana
 ARDANA_INIT_AUTO=1 /usr/bin/ardana-init</screen>
+   </step>
+   <step>
+    <para>
+     Through this process, the directory
```
Comment (dmpop) [inline xml/art_migration.xml]: This procedure creates the <filename>/srv/www/suse-12.3/x86_64/repos/PTF</filename> directory and adds it as a software repository. To enable a successful migration, you must add required Program Temporary Fix (PTF) RPM  packages to this repository manually. These packages are provide to you by be by &suse; via an active L3 support contract.

File: xml/art_migration.xml [hunk 2]
```diff
@@ -944,6 +933,14 @@ sed -i '36i\      ignore_errors: yes' _osconfig-post-upgrade.yml</screen>
      steps.
     </para>
    </step>
+   <step>
+    <para>
+     Now that the base installation has completed, the PTF packages can be removed:
```
Comment (dmpop) [inline xml/art_migration.xml]: After the base installation has been completed

File: xml/art_migration.xml [hunk 3]
```diff
@@ -1004,5 +1001,55 @@ openstack role add --user admin --project admin nova_admin</screen>
...
   </procedure>
-  </sect1>
+ </sect1>
+ <sect1>
+  <title>Applying Product Updates</title>
+  <procedure>
+   <step>
+    <para>
+     Before applying updates to your cloud, you should replace the repositories
+     configured on each node during the migration with links to an &smt; server
```
Comment (dmpop) [inline xml/art_migration.xml]: \&smt; server that mirrors the latest versions of your software

File: xml/art_migration.xml [hunk 4]
```diff
@@ -1004,5 +1001,55 @@ openstack role add --user admin --project admin nova_admin</screen>
...
+     set up to mirror the latest versions of your software.
+    </para>
+    <para>
+     The easiest way to provide the required repositories on the &lcm; Server is
+     to set up an &smt; server as described in
+     <xref linkend="app.deploy.smt_lcm"/>. Alternatives to setting up an
+     &smt; server are described in <xref linkend="cha.depl.repo_conf_lcm"/>.
+    </para>
+    <para>
+     Before continuing, you should ensure that the "Cloud" repository has been
```
Comment (dmpop) [inline xml/art_migration.xml]: Before continuing, ensure

**Comment by cadenzajon** [comment]
Thanks, Dmitri! I have incorporated your suggestions into the writeup.

**Pull request merged**